batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/services/EmployeeService.java

```diff
@@ -25,8 +40,14 @@ public Employee fetchById(Integer id) throws DataException {
     return employeeDao.fetchById(id);
   }
 
-  public Employee edit(Employee employee) throws DataException {
-    return employeeDao.edit(employee);
+  public Employee edit(Employee employee) throws DataException, ValidationException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)